src/Options/Definitions.js

```diff
@@ -11,6 +11,11 @@ module.exports.ParseServerOptions = {
     help: 'account lockout policy for failed login attempts',
     action: parsers.objectParser,
   },
+  securityChecks: {
```

Comment:
Just as reminder (because you already realized), the definitions needs to be defined in `index.js` and `buildConfigDefinitions.js` and a script will then create this file from it.